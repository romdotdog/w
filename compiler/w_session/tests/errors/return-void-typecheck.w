fn main(): u32 {
    return;
    1
}