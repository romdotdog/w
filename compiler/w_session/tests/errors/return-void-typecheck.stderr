tests/errors/return-void-typecheck.w:2:5: error: must return value
    return;
    ^^^^^^
